views/homework/extended.hbs

```diff
@@ -21,27 +21,16 @@
 		</div>
 		{{#embed "homework/components/homework_uploaded_files" deletionAllowed=false submission=this}}{{/embed}}
 		{{#unless @root.inline}}
-			{{#ifvalue teacherId value=currentUser._id}}
+			{{#if isTeacher}}
```

Comment:
this may allow just all teachers to see this, this should only be shown to current homeworks teachere instead?

Reply:
No, I adjusted the condition for `isTeacher` to be only true, if the current user is a teacher of this task